Cleanups before internal testing (#629)
0xPolygonMiden/miden-base: docs/introduction/get-started/p2p-public.md

    Accept the defaults for `Protocol`, `Node RPC Port`, and `Sqlite file path`by clicking enter.

    When `Host` comes up, enter `18.203.155.106`

    ```sh

>     Host (default: localhost): 18.203.155.106

Reviewer: Are you sure this won't change?

Author: For the next 2 weeks, I think its fine. Whatever makes the process for the users easier.

Reviewer: it's just that the docs go out of date really quickly when you're specific like this ...

Author: I promise we will change that, but for now I don't see a better solution. We will add the IP to the client soon @igamigo

    miden-client account new basic-mutable -s on-chain
    ```

>     We refer to this account as _Account C_. Note that we set the account's storage mode to `on-chain`, which means that the account details will be public and its latest state can be retrieved from the node.

Reviewer: ```suggestion
    We refer to this account as _Account C_. Note that we set the account's storage mode to `on-chain`, which means that the account details are public and its latest state can be retrieved from the node.
```

## Transfer assets between accounts

1. Now we can transfer some of the tokens we received from the faucet to our new account C. 

    To do this, from the first client run:

    ```shell
    miden-client tx new p2id <basic-account-id-A> <basic-account-id-C> <faucet-account-id> 50 --note-type public
    ```

    !!! note

>         The faucet account id is `0xad904b3138d71d3e` and can also be found on the [Miden faucet website](https://ethdenver.polygonmiden.io/) under the title **Miden faucet**.

Reviewer: ```suggestion
        The faucet account id is `0xad904b3138d71d3e` and can also be found on the [Miden faucet website](https://testnet.miden.io/) under the title **Miden faucet**.
```

Reviewer: are you sure the address is going to stay the same?

Author: For now, we will make it more generalized in the future. Now I only want quick feedback

4. Have the second account consume the note.

    ```sh
    miden-client tx new consume-notes <regular-account-ID-C> <input-note-id> 
    ```

    !!! tip

>         It's possible to use a short version of the note id: 7 characters after the `0x` is sufficient, e.g. `0x6ae613a`

Reviewer: ```suggestion
        It's possible to use a short version of the note id: 7 characters after the `0x` is sufficient, e.g. `0x6ae613a`.
```

That's it! 

> Account C will have now consumed the note and should have new assets in the account:

Reviewer: ```suggestion
Account C has now consumed the note and there should be new assets in the account:
```
